tests/no_cycles.rb

```diff
@@ -50,4 +51,9 @@ def dfs(pkg)
   dfs(pkg)
 end
 
-puts "No dependency cycles found.".lightgreen
+if @failed == 0 || @failed > 1
+  @cycles = "cycles"
+else
+  @cycles = "cycle"
+end
+puts "#{@failed} dependency #{@cycles} found.".lightgreen
```

Comment:
`abort` instead of `puts` and you don't have to delete the successful message below.

Reply:
Done.